### PR TITLE
Implement /spam issue triaging command

### PR DESCRIPTION
**Pull request by gjsjohnmurray**
This PR will enable Community Issue Trackers (currently @cleidigh, @usernamehw, @IllusionMH and myself) to label and close spam issues, which have recently become much more common.

### CODE REVIEW

**Review by gjsjohnmurray — COMMENTED** [review]
/assign @bhavyaus

**Comment by gjsjohnmurray** [comment]
Meanwhile I am finding this filter effective against the current blizzard of spam issues touting Android APK downloads (probably malicious):

```
is:issue is:open NOT APK
```

File: .github/commands.json [hunk 1]
```diff
@@ -169,6 +169,19 @@
...
+		"name": "spam",
+		"allowUsers": [
+			"cleidigh",
+			"usernamehw",
+			"gjsjohnmurray",
+			"IllusionMH"
+		],
+		"action": "close",
+		"reason": "not_planned",
+		"addLabel": "spam"
```
Comment (bhavyaus) [inline .github/commands.json:183]: @gjsjohnmurray could you please change the label to "invalid"?

Comment (gjsjohnmurray) [reply .github/commands.json]: Done

**Review by bhavyaus — APPROVED** [review]

**Review by roblourens — APPROVED** [review]
❤️

**Comment by gjsjohnmurray** [comment]
As soon as you merge this we'll be able to start using the new command.

**Comment by roblourens** [comment]
Seems like the checks aren't being picked up, maybe this was opened during the azure outage last week

**Comment by gjsjohnmurray** [comment]
Thanks for merging this @ulugbekna 

The spsmmers seem to have moved their disruptive efforts to https://github.com/microsoft/vscode-discussions/discussions, perhaps as a consequence of improved countermeasures here by GH/MS. AFAIK we don't have a bot there, but I hope something effective can be done there too ASAP.

**Pull request merged**